prepare.py

```diff
@@ -64,11 +64,31 @@ def tqdb_prepare():
     try:
         tqae_key = winreg.OpenKey(
             winreg.HKEY_LOCAL_MACHINE, LOOKUP_KEY, 0, winreg.KEY_READ)
+    except FileNotFoundError:
+        import platform
+
+        bitness = platform.architecture()[0]
+        if bitness == '32bit':
+            other_view_flag = winreg.KEY_WOW64_64KEY
+        elif bitness == '64bit':
+            other_view_flag = winreg.KEY_WOW64_32KEY
+        else:
+            raise RuntimeError("Platform architecture not recognized: " + bitness)
+
+        try:
+            tqae_key = winreg.OpenKey(
+                winreg.HKEY_LOCAL_MACHINE, LOOKUP_KEY, access = winreg.KEY_READ | other_view_flag)
+        except WindowsError as err:
+            raise RuntimeError('Could not find installation directory for Titan Quest.') from err
+
+    try:
         install = winreg.QueryValueEx(tqae_key, 'InstallLocation')[0]
-    except WindowsError:
-        print('Could not find installation directory for Titan Quest')
+    except WindowsError as err:
+        print('Could not find installation directory for Titan Quest.', err)
         return
 
+    print("Found TQ installation directory: " + install)
```

Comment:
Use the logging util here

Reply:
Can do. Only used `print` because it was being used above for the `WindowsError`. I'll change that too.